src/Compiler/Driver/GraphChecking/FileContentMapping.fs

```diff
@@ -167,7 +168,8 @@ let visitSynTypeDefnSig
             match simpleRepr with
             | SynTypeDefnSimpleRepr.Union(unionCases = unionCases) -> yield! List.collect visitSynUnionCase unionCases
             | SynTypeDefnSimpleRepr.Enum(cases = cases) -> yield! List.collect visitSynEnumCase cases
-            | SynTypeDefnSimpleRepr.Record(recordFields = recordFields) -> yield! List.collect visitSynField recordFields
+            | SynTypeDefnSimpleRepr.Record(recordFieldsAndSpreads = SynFields recordFields) ->
```

Comment:
Also here, SynFields AP drops spreads.
This is for sig files (do spreads have any special treatment in sig files?)